tmt/steps/provision/mrack.py

```diff
@@ -232,7 +232,7 @@ class BeakerGuestData(tmt.steps.provision.GuestSshData):
     hardware: Dict[str, Any] = dataclasses.field(default_factory=dict)
 
     # Provided in Beaker job
-    guestname: Optional[str] = None
+    jobId: Optional[str] = None
```

Comment:
Let's use `job_id` to be consistent with the rest of the `tmt` code (we're not using camelCase).

Reply:
fixed